### DISCUSSION TITLE
Rendering fails on Windows 11

### DISCUSSION THREAD

**Discussion by cyrilby** [GetPublii/Publii#1535] [Help]
Hi, I've been using Publii to create several pages and everything was working fine until all of a sudden, it started showing "Rendering fail" errors, always at 80%:

![image](https://github.com/GetPublii/Publii/assets/102741768/e6b84e4f-47f2-4a49-8d26-eff314d0e56c)

I've gotten the error both when it comes to `author` pages and `tag` pages. I've been previously able to resolve it by deleting a tag or an author but this is unsustainable. Reinstalling Publii didn't fix this, neither did changing to a different theme or deleting the `preview` folder within the website.

Any ideas as to what I can try next so as to resolve this?

**Comment by dziudek** [comment]
Hi,

Do you have that folder at all? Or it does not exist?

**Comment by cyrilby** [comment]
Hi,

Yes, I have the folder and it is actually completely empty. I can easily delete it from Windows Explorer. After deleting it, I still get the same error in Publii.

**Comment by slymeasy** [comment]
I have this same error. When I delete all the folders in the media folder and the replace them [with an empty folder] it syncs properly:
- posts
- tags
- files
- authors

**But then the next time I try to sync I get the error again.**

**Comment by dziudek** [comment]
@slymeasy @cyrilby I suspect that npm module used to delete empty folders causes this issue - in v.0.46.2 planned for next week it will be replaced by our custom code which should handle EPERM errors in a different way which shouldn't cause such issues
